### DISCUSSION TITLE
Cargo-64: Cargo-16 remix

### DISCUSSION THREAD

**Discussion by MirceaKitsune** [redeclipse/base#1482] [Maps and Content Showcase]
After months of constant work, I'm happy to finally have a sharable version of my fixes to Cargo16. RE's Cargo16 just like UT's Deck16 is one of my favorite maps: For a long time I've wanted to create my own remake... though the base 2.0 port is great, the map has a ton of issues that had to be corrected in every department (geometry, texturing, entities, etc). I ended up remaking almost everything with results I'm very happy of!

The hallmark of my version is the differences between the Default and Alternate modes: Default is meant to resemble UT2004's Deck17, lit primarily by a red sun and using red slime in a fiery environment, whereas Alternate aims to represent UT99's Deck16 with green slime and focus on interior lighting. A lot of details differ or switch colors between red and green based on the selected mode, including the warning pattern at the base of some walls!

To separate it from the old map and make it less obvious where it's inspired from, I renamed it to Cargo-64: I figured the switch from 16 to 64 would symbolize the upgrade from the 16-bit to the 64-bit era.

Give it a try and let me know what you think! Below are my versions as I will update them, as well as screenshots comparing the visuals between the two modes. Once it's ready this is intended to replace Cargo-16 in Git.

![20240225_191153](https://github.com/redeclipse/base/assets/825418/2c8dea2c-bc9d-43fa-862c-35774984ce0f)
![20240225_191205](https://github.com/redeclipse/base/assets/825418/eed17a9a-b496-4986-98fb-5d2ffbe704b1)
![20240228_032817](https://github.com/redeclipse/base/assets/825418/323ce6e8-3318-4b08-89e0-60c2e8fc6faf)
![20240228_032848](https://github.com/redeclipse/base/assets/825418/caf500d7-4c7f-4dcc-8953-7bfa09bf6fef)
![20240228_055631](https://github.com/redeclipse/base/assets/825418/fadbbb77-45b3-4136-8297-ce38aa88a57e)
![20240228_055801](https://github.com/redeclipse/base/assets/825418/45458fdb-290e-43d2-92e1-8c52031f4aab)
![20240301_011457](https://github.com/redeclipse/base/assets/825418/bc6f8abe-a923-4f77-8861-24a27ae49698)
![20240301_011610](https://github.com/redeclipse/base/assets/825418/65186a3e-c80c-49ef-b51a-4331ccc51532)

- [cargo64_r2336.zip](https://github.com/redeclipse/base/files/14859832/cargo64_r2336.zip) Initial version, intended for testing only. A few things still need to be fixed, such as adding proper sounds and updating bot waypoints. More tweaks to smaller details likely.
- [cargo64_2617.zip](https://github.com/redeclipse/base/files/15050149/cargo64_2617.zip) Many fixes in all areas: Geometry, texturing, lighting, sound. Placed ambient sounds and sound enviornment entities. The radius of most lights has been reduced, fixing excess brightness and slightly improving performance. Teleporter tunnel improvements. Added AI waypoints for all ground surfaces, crate jumps are currently impossible to path due to #1500 causing bad waypoints to be placed in random locations.

**Comment by MirceaKitsune** [comment]
More updates and fixes underway. Here are some better shots in the meantime: Some of the original screenshots are of older versions or don't capture all of the areas.

![20240404_201305](https://github.com/redeclipse/base/assets/825418/ce393df7-7afa-4198-a7c6-29fca7cd2942)
![20240404_201324](https://github.com/redeclipse/base/assets/825418/e3a7b3c1-fe2b-465f-a7c0-1b78220a180a)
![20240404_204823](https://github.com/redeclipse/base/assets/825418/247a1e40-5d06-44e5-876f-18dec1006ab6)
![20240404_204911](https://github.com/redeclipse/base/assets/825418/f019308c-be7c-4459-8907-132467e7e48b)
![20240404_202837](https://github.com/redeclipse/base/assets/825418/28b6f08c-64d2-4e2d-8deb-97c492623c9c)
![20240404_202922](https://github.com/redeclipse/base/assets/825418/5b8b5eda-bf47-4f85-9e0b-684c5b24397b)
![20240404_205105](https://github.com/redeclipse/base/assets/825418/3612ceed-16bd-430d-8aa6-7d5b39c6e01b)
![20240404_205136](https://github.com/redeclipse/base/assets/825418/32fc0a6b-826b-4c88-bfc9-5dddda53f384)

**Comment by Jigoku** [comment]
Forgot to comment on this before when i did look at it, sorry about that! 
Really liking how this is looking👍 Especially with the dynamic variant themes
